openstack/02b-MAChigh/neighbors.c

```diff
@@ -543,7 +543,8 @@ void neighbors_updateMyDAGrankAndNeighborPreference() {
             rankIncrease = (uint16_t)(rankIncreaseIntermediary >> 10);
          }
          
-         tentativeDAGrank = neighbors_vars.neighbors[i].DAGrank+rankIncrease;
+         tentativeDAGrank = neighbors_vars.neighbors[i].DAGrank;
+         tentativeDAGrank += rankIncrease;

```

Comment:
@koalo yours is good! I just didn't see the type of the two variables. 
Congratulations on finding this! I will merge this code after the build passed :)